src/java.base/share/classes/sun/security/x509/OtherName.java

```diff
@@ -89,8 +89,11 @@ public OtherName(DerValue derValue) throws IOException {
         DerInputStream in = derValue.toDerInputStream();
 
         oid = in.getOID();
-        DerValue val = in.getDerValue();
-        nameValue = val.toByteArray();
+        DerValue derValue1 = in.getDerValue();
+        if (!derValue1.isContextSpecific((byte)0)) {
```

Comment:
Where do you check for constructed (0x20)?

Reply:
Oops, I should add an `isConstructed()` check.